docs/data/data-grid/getting-started/getting-started.md

```diff
@@ -145,13 +145,13 @@ Please pay attention to the license.
 
 ### Plans
 
-The component comes [in different plans](/pricing/):
+The component comes [in different plans](https://mui.com/pricing/):
```

Comment:
So that once this content is not longer the active version, developers will still land on the latest pricing page.